includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -56,19 +56,23 @@ public function __construct() {
 		add_action( 'woocommerce_stripe_updated', array( $this, 'verify_domain_if_needed' ) );
 		add_action( 'update_option_woocommerce_stripe_settings', array( $this, 'verify_domain_on_new_secret_key' ), 10, 2 );
 
+		add_action( 'init', array( $this, 'add_domain_association_rewrite_rule' ) );
+		add_filter( 'query_vars', array( $this, 'whitelist_domain_association_query_param' ), 10, 1 );
+		add_action( 'parse_request', array( $this, 'parse_domain_association_request' ), 10, 1 );
+
 		$this->stripe_settings         = get_option( 'woocommerce_stripe_settings', array() );
 		$this->stripe_enabled          = $this->get_option( 'enabled' );
 		$this->payment_request         = 'yes' === $this->get_option( 'payment_request', 'yes' );
 		$this->apple_pay_domain_set    = 'yes' === $this->get_option( 'apple_pay_domain_set', 'no' );
+		$this->apple_pay_rewrite_flush = 'yes' === $this->get_option( 'apple_pay_rewrite_flush', 'no' );
```

Comment:
Could we make all these something like `apple_pay_rw_rules_flushed` instead? To make it easier to read here and below and think "if this is true, this means the rewrite rules have been flushed"

Reply:
Removed that flag, since the rules are now being flushed just in time when needed.